fecgov/openFEC
# [do not merge before PR# 4610] add disclosure.dim_cmte_ie_inf table and enable data sync with source database

## Summary (required)

- Resolves #4574 Add DISCLOSURE.DIM_CMTE_IE_INF to Aurora databases
In order to captures instances of PCC to PAC Conversions, we need data from DISCLOSURE.DIM_CMTE_IE_INF.  Currently this table is not in Aurora databases.   disclosure.dim_cmte_ie_inf need to be added to the Aurora databases and keep sync with its couterpart data source in Oracle database.
- This table had been created in all Aurora databases since data sync will happened to all FEC Aurora databases.  The migration file is submitted to keep official database structure in sync.  

## How to test the changes locally

- Download the branch to local machine
- Run flyway migrate (or invoke create_sample_db), make sure it finish successfully.  
- Log in local database, make sure table disclosure.dim_cmte_ie_inf had been created
- In local database, 
delete from flyway_schema_history where version = '0212';
- run flyway migration again, make sure no error occurs

## Impacted areas of the application
List general components of the application that this PR will affect:

-  None

Required Reviewer:  @hcaofec 
Optional Reviewer: @fec-jli 

## Related PRs
List related PRs against other branches:

branch | PR
------ | ------
fix/other_pr | [link]()
feature/other_pr | [link]()
